addons/l10n_id/models/res_bank.py

```diff
@@ -27,8 +28,8 @@ def _l10n_id_make_qris_request(endpoint, params):
 class ResBank(models.Model):
     _inherit = "res.partner.bank"
 
-    l10n_id_qris_api_key = fields.Char("QRIS API Key", groups="base.group_system")
-    l10n_id_qris_mid = fields.Char("QRIS Merchant ID", groups="base.group_system")
+    l10n_id_qris_api_key = fields.Char("QRIS API Key")
+    l10n_id_qris_mid = fields.Char("QRIS Merchant ID")
```

Comment:
I found out about this error while writing unit test.
If I make a PoS user (with non-admin rights), there will be access rights error when trying to verify the payment status (method: _l10n_id_qris_fetch_status) because both fields are being accessed.

I think in this case, we can do the `groups="base.group_system"` on the view instead?

Reply:
> I think in this case, we can do the groups="base.group_system" on the view instead?
This would still leave the field vulnerable. Better keep the group and maybe add a sudo() just before reading the fields when doing an API call